test/routes/userAdminTest.js

```diff
@@ -49,16 +49,6 @@ describe('/admin', () => {
     })
   })
 
-  describe('GET /admin', () => {
-    it('redirects to ro user list', () => {
-      const app = createApp({ userAdminServiceStub: userAdminService }, 'batchUser')
-      return request(app)
-        .get('/admin/')
-        .expect(302)
-        .expect('Location', '/admin/roUsers')
-    })
-  })
-
```

Comment:
Same comment as above - just limited data used in the test cases. If one of queries was more 'open' than it ought to be there is a chance that the test still passes but in real environments the additional data could cause it to fail.

Reply:
Added more cases to make it more explicit that any input is accepted